src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -172,6 +172,18 @@ function EmailsList({ summaries, auth }) {
   return null;
 }
 
+function getTextColorForHoursLogged(hoursLogged, promisedHours) {
+  const percentage = (hoursLogged / promisedHours) * 100;
+
+  if (percentage < 49) {
+    return 'red';
+  } else if (percentage >= 50 && percentage < 100) {
+    return 'lightgreen'; // Bright green color
+  } else {
+    return 'black'; // Default color
+  }
+}
+
```

Comment:
49 <= `percentage` < 50 case is not covered, therefore it becomes black.

Reply:
Hi, thanks for discover this bug. I have change the logic of this function. You can test it again. Thank you so much!